rust-escrow/stable-escrow/src/escrow.rs

```diff
@@ -469,10 +479,20 @@ impl Escrow {
         .to_string()
         .into_bytes()
     }
+
+    fn measure_account_storage_usage(&mut self) {
+        let initial_storage_usage = env::storage_usage();
+        let tmp_account_id = AccountId::new_unchecked("a".repeat(64));
+        self.deposits.insert(&tmp_account_id);
+        self.ft.accounts.insert(&tmp_account_id, &0u128);
+        self.account_storage_usage = env::storage_usage() - initial_storage_usage;
+        self.ft.accounts.remove(&tmp_account_id);
+        self.deposits.remove(&tmp_account_id);
+    }
```

Comment:
This is new to me. What are we doing exactly? Why create a temp account, then remove it?
I see we are "faking" operations to estimate storage that then we'll charge to the user?

Reply:
Yes that is the idea. That method calculates the space that a user is going to use.  this helps us to later calculate the storage that a user must pay to use the contract